docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+services:
+    edagames-web:
+        build: .
+        ports:
+            - "8000:8000"
+        volumes: 
+            - .:/edagames-django
+        environment: 
+            - SECRET_KEY=2g&=8-j*xzb7^b*ej3&hd@=s@hfh64xj4x$wvn1^xjl3bv7io4
+            - SECRET_KEY_JWT=EDAGame$$!2021
+            - SOCIAL_AUTH_FACEBOOK_KEY=1194039207691875
+            - SOCIAL_AUTH_FACEBOOK_SECRET=f469bcead34a7766f1fa01f81b040ddf
```

Comment:
This should be env variables

Reply:
You can use .env files check https://docs.docker.com/compose/environment-variables/

Reply:
Done! Thank you nico.